Dockerfile

```
@@ -1,41 +1,113 @@
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+###############################################################################
+# 3. Runtime Stage - minimal size
+###############################################################################
+FROM debian:bookworm-slim AS runtime
 
 WORKDIR /app
-COPY --from=build /app/_build/prod/rel/wanderer_kills ./
 
+# Install runtime dependencies in one layer
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)